Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour
+{
+    BoxCollider2D box;
+    
+    GameObject[] EnemyBandits;
```

Comment:
explicitly specify `private` (do this for all of the private fields and methods).